learning/courses/basics-of-quantum-information/index.mdx

```diff
@@ -38,4 +40,16 @@ To make the most out of this course, we recommend familiarity with basic linear
 
 - John K. Hunter. [An Introduction to Real Analysis: Chapter 1](https://www.math.ucdavis.edu/~hunter/intro_analysis_pdf/ch1.pdf)
 
-   The first chapter of these lecture notes includes a more formal and detailed introduction to sets and functions.
\ No newline at end of file
+   The first chapter of these lecture notes includes a more formal and detailed introduction to sets and functions.
+
+## Exam
+
+To earn your Basics of quantum information badge, take the exam at IBM Training. This exam is intended to be taken after reading the lessons in this course. After you pass the exam, you will be notified by Credly that you have earned a badge.
```

Comment:
It will translate a lot better to say

```suggestion
To earn your badge for Basics of quantum information, take the exam at IBM Training. This exam is intended to be taken after reading the lessons in this course. After you pass the exam, you will be notified by Credly that you have earned a badge.
```

Other languages struggle with how English uses phrases as adjectives before the noun.

Ditto the rest of the courses